website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1097,9 +1103,42 @@ $ curl \
+
+It is considered "unsafe" to delete workspaces with resources under management, as deleting a workspace on Terraform Cloud will leave all infrastructure managed by the workspace orphaned from Terraform.
+
+A workspace can be safe deleted via two endpoints, which behave identically. One refers to a workspace by its ID, and the other by its name and organization.
+
+`POST /workspaces/:workspace_id/actions/safe-delete`
 
-A workspace can be deleted via two endpoints, which behave identically. One refers to a workspace by its ID, and the other by its name and organization.
+| Parameter       | Description                       |
+| --------------- | --------------------------------- |
+| `:workspace_id` | The ID of the workspace to delete |
+
+`DELETE /organizations/:organization_name/workspaces/:name/actions/safe-delete` 
+
+| Parameter            | Description                                                                                 |
+| -------------------- | ------------------------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization the workspace belongs to.                                      |
+| `:name`              | The name of the workspace to delete, which can only include letters, numbers, `-`, and `_`. |
+
+| Status  | Response                  | Reason(s)                                                             |
+| ------- | --------------------------| --------------------------------------------------------------------- |
+| [204][] | Nothing                   | Successfully deleted the workspace                                    |
+| [404][] | [JSON API error object][] | Workspace not found, or user unauthorized to perform workspace delete |
+| [409][] | [JSON API error object][] | Workspace is not safe to delete                                       |
+
+## Force Delete a workspace
+
+Force deleting a workspace removes it without checking whether the workspace currently has [resources under management](/cloud-docs/workspaces/settings#deleting-a-workspace-with-resources-under-management). This ignores all of the safety checks present in a [safe delete](#safe-delete-a-workspace).
+
+By default workspaces may only be force deleted by an organization owner. If the [Teams can delete workspaces with resources under management](/cloud-docs/users-teams-organizations/organizations#general) organization setting is enabled, workspace admins may also force delete.
+
+!> **Warning:** Use caution when force deleting a workspace. Any infrastructure managed by the workspace will no longer be managed by Terraform. It is recommended that you apply a [Destroy Run](/cloud-docs/run/modes-and-options#destroy-mode) before deleting a workspace.
+
+A workspace can be force deleted via two endpoints, which behave identically. One refers to a workspace by its ID, and the other by its name and organization.
```

Comment:
Is this also for workspaces that are locked? I thought I saw that somewhere else in this docs draft :) 

```suggestion
During a force delete, Terraform Cloud removes the specified workspace without checking whether it is managing resources. We recommend using the [safe delete endpoint](#safe-delete-a-workspace) instead, when possible.

!> **Warning:** Terraform cannot track or manage the workspace's infrastructure after deletion. We recommend [destroying the workspace's infrastructure](/cloud-docs/run/modes-and-options#destroy-mode) before you delete it. 

By default, only organization owners can force delete workspaces. Organization owners can also update [organization's settings](/cloud-docs/users-teams organizations/organizations#general) to let workspace admins force delete their own workspaces.

You can use two endpoints to force delete a workspace, which behave identically. One endpoint identifies the workspace with its workspace ID and the other endpoint identifies the workspace with its name and organization.
```